network/prefetchAllActiveForecasts.ts

```diff
@@ -57,12 +55,14 @@ export const prefetchAllActiveForecasts = async (
   }
 
   const endDate: Date = currentDateTime;
-  const startDate = add(endDate, DEFAULT_OBSERVATIONS_WINDOW);
   if (center_id === 'NWAC') {
-    void NWACObservationsQuery.prefetch(queryClient, nwacHost, center_id, startDate, endDate, logger);
+    // NWAC fetches in pages of 50 working backwards from endDate
```

Comment:
@floatplane the days of the NWAC obs API might be numbered but if you'd like I can try to make it do better with 2wk periods if it would make the rest of our code easier to grok here

Reply:
I think it's pretty reasonable to leave it alone - in both cases we prefetch a chunk of results, I can't really picture a scenario where anyone notices the difference in paging behavior!